model/publishing/delivery/listeners/DeliveryEventsListeners.php

```diff
@@ -90,4 +95,18 @@ protected static function checkActions($name)
         }
     }
 
+    /**
+     * @param \core_kernel_classes_Resource $delivery
+     * @return bool
+     */
+    protected static function checkSyncProperty(\core_kernel_classes_Resource $delivery)
```

Comment:
Please rename this method to something like `mustBePublished($delivery)` or `isDeliverPublishable($delivery)` and add php doc with method description.